src/main/java/ui/Ui.java

```diff
@@ -47,7 +47,7 @@ public void showError(String message) {
     }
 
```

Comment:
Furthermore, to be picky again, from the code style page, they state that all public methods and classes require Javadoc comments.